python/tests/plugins/test_gen_design_doc_impl.py

```diff
@@ -164,3 +181,16 @@ def assert_use_case(self, markdown: str) -> None:
       target: model1
       action: move from point beta back to point alpha
 """
+TEST_MODEL_2 = """
```

Comment:
I forgot to mention this while we were working on it but, while we did this to get things working, it would probably be better to use our `create_schema_definition` helper function to create these schemas programmatically so we don't have to store several different constants far from where they're used.

Reply:
I am a little apprehensive on that currently. I do think that would be beneficial. But I am not wanting to break anything and causing more delay in getting this added. If it is required I can definitely look into getting those constants removed and exploring the `create_schema_definition` option.

What are your thoughts?

Reply:
That's fine, we can hold off on switching. Instead, please create a story with the label "chore" that addresses updating those test models.

Reply:
created issue #421